src/views/ajax.js

```diff
@@ -5,6 +5,36 @@ const arrowIcon = require('./open').arrowIcon;
 const icons = require('../assets/icons').default;
 const statusIcon = require('../assets/icons').statusIcon;
 
+/**
+ * removeOrigin - function to remove origin from a URL.
+ *
+ * @param {String} url URL to remove the origin from.
+ * @param {String} origin Origin to remove from URL.
+ */
+const removeOrigin = (url = '', origin = window.location.origin) => {
+  if (url.substr(0, origin.length) === origin) {
+      url = url.substr(origin.length, url.length);
+  }
+
+  return url;
+};
+
+/**
+ * removeOriginFromUrl - function to remove origin from a URL
+ *                       reguarding `http` and `https`.
+ *
+ * @param {String} url URL to remove the origin from.
+ * @param {String} origin Origin to remove from URL.
+ */
+const removeOriginFromUrl = (url = '', origin = window.location.origin) => {
+  url = url.trim();
+
+  url = removeOrigin(url, origin.replace(/^http\:\/\//, 'https://'));
```

Comment:
Does this logic match what we do in the client to determine when to strip off the origin?  IIRC, the client will not strip if the scheme is different.